src/pages/event-attendees/event-attendees.ts

```diff
@@ -56,6 +57,25 @@ export class EventAttendeesPage {
     });
   }
 
+  public doRefresh(refresher) {
+    this.attendeesService.loadAttendees(this.event.id).subscribe(
```

Comment:
@niranjan94 Avoided duplication. But not using callback. Please check if the approach is fine.